_includes/markdown/Vue.md

```diff
@@ -0,0 +1,76 @@
+The purpose of this document is to help guide you through working with a potential project using the JavaScript framework, Vue. Outside of The [Official Vue Documentation](https://vuejs.org/), you should find here personalized recommendations for using the framework based on the types of projects we typically see at 10up. We have broken these recommendations out into common elements we tend to interact with often in client work. If something you’re looking for isn’t represented you can either submit a pull request to this repository or refer to the [Official Vue Documentation](https://vuejs.org/v2/guide/).
+
+For Vue-specific coding style and pattern recommendations please see the official [Vue style guide](https://vuejs.org/v2/style-guide/).
+
+## Figuring Out If Vue Is Right For Your Project
```

Comment:
@timwright12 @saltcod I think we need to redo this section.

I am still unclear on the React versus Vue piece. The sentence: "If you're answering "no" to some of those questions, but still feel a JavaScript-framework solution is the correct model, you may want to read through our [React]({{ site.baseurl }}/react) recommendations as a path forward for a larger scale application." makes very little sense to me. If my project doesn't contain a lot of components that need to interact with an API, React is a better fit?

Does React allow for client side templating at all? Seems like the reasons to use Vue are client side templating and lighter library.

Reply:
@tlovett1 Terry ( @saltcod ) and I are going to take this offline and rework the section. It shouldn't be a this vs. that kind of thing. They should be independently evaluated for the benefit of the project. Ultimately before you even use one you need to answer the question: "Do I need a JS framework at all" - so those parts will be the same in each document. Beyond that there should just be a couple things that nudge you towards one and away from the other. But I want to get away from the angle/perception that the presence of a Vue doc is anti-React and vice versa.